src/ameba/rule/lint/empty_loop.cr

```diff
@@ -58,7 +56,9 @@ module Ameba::Rule::Lint
 
     private def check_node(source, node, loop_body)
       body = loop_body.is_a?(Crystal::Block) ? loop_body.body : loop_body
-      issue_for node, MSG if body.nil? || body.nop?
+      return unless body.nil? || body.nop?
+
+      issue_for node, MSG
```

Comment:
could you elaborate on why do we favor early return here over the if statement, but do the opposite in the method above 😄

Reply:
Haha, sure. In cases of one simple expression I've opted to attaching them as trailing expressions, whereas in cases there's more statements in the method body I've used the early return. Also, notice there's already `test` method definition with trailing condition.